app/models/module_item.rb

```diff
@@ -154,6 +154,12 @@ def page_name
   # @return [Boolean]
   delegate :valid?, to: :model
 
+  def notes?
+    model.notes?
+  rescue StandardError
```

Comment:
What's the alternative to the `rescue`? If the model is a `Questionnaire` it shouldn't, is it for video pages not having this method.

Reply:
it's a catchall - any page type, now or in the future, which doesn't have notes

Reply:
Could we use `NoMethodError` instead so the catchall won't swallow everything, or preferably maybe `model.respond_to?(:notes?)` in a ternary so it is explicit.